llvm/test/Transforms/LoopVectorize/memdep-fold-tail.ll

```diff
@@ -24,65 +24,17 @@ target datalayout = "e-p:64:64:64-i1:8:8-i8:8:8-i16:16:16-i32:32:32-i64:64:64-f3
 define void @maxvf3() {
 ; CHECK-LABEL: @maxvf3(
 ; CHECK-NEXT:  entry:
-; CHECK-NEXT:    br i1 false, label [[SCALAR_PH:%.*]], label [[VECTOR_PH:%.*]]
```

Comment:
This looks like a regression? The maximum safe VF should be 2 in this case I think, as the dependence isn't classified as preventing load-store forwarding?